component/loki/source/azureeventhubs/internal/parser/parser.go

```diff
@@ -0,0 +1,197 @@
+package parser
+
```

Comment:
I think this was copied from Promtail, right? Could we please add a note like the following to files we copied over? We do this for other packages we're forking to keeping track of what we've forked over.


```suggestion
// This code is copied from Promtail. The parser package is used to
// enable parsing entries from Azure Event Hubs  entries and forward them
// to other loki components.

```